generator/templates/client/parameter.gotmpl

```diff
@@ -235,7 +235,7 @@ func ({{ .ReceiverName }} *{{ pascalize .Name }}Params) WriteToRequest(r runtime
   {{ end }}
 
   {{if .IsBodyParam}}
-  {{ if or .Schema.IsInterface (and .Schema.IsNullable (not .HasDiscriminator)) }}if {{ .ValueExpression }} != nil {
+  {{ if or .Schema.IsInterface (and .Schema.IsArray .Child .Child.IsInterface) (and .Schema.IsNullable (not .HasDiscriminator)) }}if {{ .ValueExpression }} != nil {
```

Comment:
I think that may need to be `.Schema.Child`

Reply:
There are other references in the file to `.Child`. I get an error with `.Schema.Child`
`at <.Schema.Child>: can't evaluate field Child in type *generator.GenSchema`